Devrel (#90)
argoapp-live/argo-docs: versioned_docs/version-3.0.0-beta/compute/getting-started.md

---
id: getting-started
title: Getting Started
sidebar_label: Getting Started 
slug: /compute/getting-started
---

> ### H3 Spheron Compute 

Reviewer: ```suggestion
### Spheron Compute 
```

Compute refers to the processing power and resources required to perform computational tasks such as data processing, simulations, and algorithms. In computing, compute is often measured in terms of CPU cycles, memory, and storage. The amount of compute required for a particular task is determined by the complexity of the task and the size of the data being processed

A Compute Instance is a virtual machine that can be used to host applications and services. This document provides step-by-step instructions for creating a Compute Instance using the Akash's cloud computing platform. 

## Prerequisites
1. A Spheron account with project set up.
2. Dockerised image to create an instance. 

:::info

Here is how **Create a docker image** and push it to Dockerhub. Check [this link here][def].

:::

## Steps
1. Open the Spheron dashboard in your browser and navigate to the Compute section.
2. Click on the "Create New Cluster" button to start creating your instance.
3. On the "Create a new project" page, you will need to configure the following:
- Import form Dockerhub 
- Start from a new template

<img src={require('@site/static/img/compute-1.png').default} />

> ## 	Add Organization Name and Username 

Reviewer: ```suggestion
## Create your Organization
```

Reviewer: Btw I don't understand the context change here. Before this, we are talking about how to create a cluster but now it changed to creating a new org. I don't think it make a lot of sense to me when I read thru get started doc

> Add Organisation name and the username aslo you can add organisation avatar and then click on save. 

Reviewer: ```suggestion
You need to add the Organisation name and the username. Also, you can add an avatar of your liking and then click on Save. 
```

> <img src={require('@site/static/img/addorg.png').default} />

Reviewer: remove this one too

## Create new cluster 

> Click to create a new cluster, and the organization name will be reflected on the top left corner. 

Reviewer: We need to explain terminology here. What does it mean by Cluster or just give the user a simple step-by-step walkthrough of how to deploy on compute without the requirement?

<img src={require('@site/static/img/newcluster.png').default} />

## Create a new project 

> static/img/createnew.png

Reviewer: what's this?

> <img src={require('@site/static/img/createnew.png').default} />

Reviewer: What is your understanding of this screenshot? Is it enough to pass on a screenshot without giving context over what they need to choose or which option is for which use case?
